source/operations/install-deploy-manage/deploy-minio-tenant-helm.rst

```diff
@@ -182,8 +246,78 @@ This method may support easier pre-configuration of the Tenant compared to the :
 
    Each chart contains a ``values.yaml`` file you can customize to suit your needs.
    For details on the options available in the MinIO Tenant ``values.yaml``, see :ref:`minio-tenant-chart-values`.
```

Comment:
Is this link still needed if we are now actually discussing the contents of values.yaml here?

Reply:
I think so - `values.yaml` has a lot more gritty detail, this tutorial (for now) is very high level.